client/landing/jetpack-cloud/layout.js

```diff
@@ -25,7 +26,7 @@ class JetpackCloudLayout extends Component {
 						{ this.props.secondary }
 					</div>
 					<div id="primary" className="layout__primary">
-						{ this.props.primary }
+						{ this.props.primary || <EmptyContent /> }
```

Comment:
This is a bit weird. Could we set `context.primary` to an `<EmptyContent />` earlier if we're running into a 404?

Reply:
This code pathway isn't actually ever called if we hit a 404, that's handled much higher up, in the Express layer. This is here to be imported as the styles weren't being part of our bundle.  I agree there could be a better way, but honestly I think we're going to use this component once we build it out anyway.